src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/RoboticsCallbackHandler.java

```diff
@@ -79,6 +84,20 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             log.info("Is case valid to send to robotics {} for case id {}", isCaseValidToSendToRobotics, callback.getCaseDetails().getId());
 
             if (isCaseValidToSendToRobotics) {
+                if(gapsSwitchOverFeature){
```

Comment:
You should be able to move this block outside of isCaseValidToSendToRobotics. I don't think we want to bother with Robotics at all if we have the feature and the route is LA. I would double check with Andrew, though.